pom.xml

```diff
@@ -1,349 +1,350 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
It is not possible to see changes here because the whole file has been reformatted. Please use 3-space indentation in XML files as well. It is defined in our code style.